### DISCUSSION TITLE
Ant Design的icon图标就这么多吗 感觉有点不够用

### DISCUSSION THREAD

**Discussion by 2010363329** [ant-design/ant-design#46280] [Q&A]
网站链接：https://ant-design.antgroup.com/components/icon-cn
有没有什么比较好的 强大 稳定 且样式风格跟ant desigb差不多的
希望ant多加点icon 感觉不够用呀 呜呜呜～

**Comment by afc163** [comment]
你还缺什么 icon 呢？

**Reply by 2010363329** [reply]
我我我 哈哈 这怪不好意思的（但是真的想要几个图标 一个信箱的图标 类似这样的：📪 当然可以做的更好看的）
目前只发现了个mail图标 好像跟主题信箱不太匹配

**Reply by yoyo837** [reply]
业务上的，建议自己团队画，iconfont挂上去

**Reply by 2010363329** [reply]
自己的小网站 就我一个人 呜呜呜～

**Comment by arvinxx** [comment]
> 网站链接：https://ant-design.antgroup.com/components/icon-cn 有没有什么比较好的 强大 稳定 且样式风格跟ant desigb差不多的 希望ant多加点icon 感觉不够用呀 呜呜呜～

建议使用 https://lucide.dev/  。去他们 github 的 pr 看看，这图标做的才叫专业:

https://github.com/lucide-icons/lucide/pull/1538

<img width="989" alt="image" src="https://github.com/ant-design/ant-design/assets/28616219/646eaaed-4f80-4d2a-b07f-648a4f3fd5a9">

**Reply by Wxh16144** [reply]
我就知道会有你 [doge]

**Reply by 2010363329** [reply]
好滴 感谢 已收藏

**Reply by crazyair** [reply]
@arvinxx 怎么没有中文版

**Reply by 2010363329** [reply]
外国的 你图标要啥中文 直接引用不就好了
